app/javascript/controllers/category_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus"
+import Tags from "bootstrap5-tags"
+
+export default class extends Controller {
+  static targets = ["field"]
+
+  addCategory(event) {
+    const vals = Tags.getInstance(this.fieldTarget).getSelectedValues()
+
+    if (!vals.includes(event.target.value)) {
+      Tags.getInstance(this.fieldTarget).addItem(event.target.value)
```

Comment:
Noticed one small bug - if the tags are initially empty and you add one, the field label doesn't float successfully.

I think you either need to fire a `changed` event on it... or remove the `form-control-blank` class?